op-e2e/setup.go

```diff
@@ -624,12 +628,20 @@ func (cfg SystemConfig) Start(t *testing.T, _opts ...SystemConfigOption) (*Syste
 		return nil, err
 	}
 
-	for name := range cfg.Nodes {
-		if name == RoleL1 {
-			return nil, fmt.Errorf("node name %s is reserved for L1 node", RoleL1)
-		}
+	// Ordered such that the Sequencer is initialized first. Setup this way so that
+	// the `RollupSequencerHTTP` GethOption can be supplied to any sentry nodes.
+	l2Nodes := []string{RoleSeq, RoleVerif}
+	for _, name := range l2Nodes {
 		var ethClient services.EthInstance
 		if cfg.ExternalL2Shim == "" {
+			if name != RoleSeq && !cfg.DisableTxForwarder {
+				cfg.GethOptions[name] = append(cfg.GethOptions[name], func(ethCfg *ethconfig.Config, nodeCfg *node.Config) error {
+					ethCfg.RollupSequencerHTTP = sys.EthInstances[RoleSeq].UserRPC().RPC()
+					cfg.Loggers[name].Warn("URL OPTION SET", "url", ethCfg.RollupSequencerHTTP)
```

Comment:
The `if` statement is hard to parse due to the amount of boolean logic in it. If its not the sequencer and its not disable tx forwarder, this is saying if its a verifier and forwarding is enabled. Just typing this out to double check. I also think including the role name in the logline is nice

Reply:
yea that's right. Also the log line was included by accident when debugging why this code change wasn't working. I'm going to remove it as I dont think it's particularly useful